src/Tickets/Emails/Email/Purchase_Receipt.php

```diff
@@ -115,38 +68,14 @@ public function get_title(): string {
 	}
 
 	/**
-	 * Get email heading.
+	 * Get default recipient.
 	 *
 	 * @since TBD
 	 *
-	 * @return string The email heading.
+	 * @return string
 	 */
-	public function get_heading(): string {
-		$heading = tribe_get_option( static::$option_heading, $this->get_default_heading() );
-
-		/**
-		 * Allow filtering the email heading for Purchase Receipt.
-		 *
-		 * @since TBD
-		 *
-		 * @param string          $heading  The email heading.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
-		 */
-		$heading = apply_filters( 'tec_tickets_emails_purchase_receipt_heading', $heading, self::$id, $this->template );
-
-		/**
-		 * Allow filtering the email heading globally.
-		 *
-		 * @since TBD
-		 *
-		 * @param string          $heading  The email heading.
-		 * @param string          $id       The email id.
-		 * @param Tribe__Template $template Template object.
-		 */
-		$heading = apply_filters( 'tec_tickets_emails_heading', $heading, self::$id, $this->template );
-
-		return $this->format_string( $heading );
+	public function get_default_recipient(): string {
+		return '{purchaser-email}';
```

Comment:
Not sure if this should come from a placeholder, as we would only use placeholders for "printable" chunks (subject, title, additional content).

In any case the format is with underscores `{purchaser_email}`. We could probably define this afterwards, when we properly populate the email with the context data.

```suggestion
		return '{purchaser_email}';
```

Reply:
Yeah. I figured we'd be changing this when we start implementing the data that goes into it. I just wanted to put in a placeholder for now.